app/lib/clients/bambooHR_client.rb

```diff
@@ -0,0 +1,36 @@
+def find_employees
+  client = Bamboozled.client(subdomain: AppEnv['BAMBOOHR_SUBDOMAIN'], api_key: AppEnv['BAMBOOHR_API_KEY'])
+
+  day_before_yesterday = (Time.zone.yesterday - 1).to_formatted_s(:iso8601)
+  yesterday = Time.zone.yesterday.to_formatted_s(:iso8601)
+  today = Time.zone.now.to_formatted_s(:iso8601)
+  tomorrow = Time.zone.tomorrow.to_formatted_s(:iso8601)
+  day_afer_tomorrow = (Time.zone.tomorrow + 1).to_formatted_s(:iso8601)
+
+  time_off = client.time_off.whos_out(Time.zone.now, Time.zone.now)
+  employees_info = client.report.custom(%w[id birthday hireDate isPhotoUploaded])
+  employees = []
+
+  employees_info.each do
+    if employee['birthday'].present? &&
+      [day_before_yesterday, yesterday, today, tomorrow, day_afer_tomorrow].any? { |d|
+        if d.include? employee['birthday']
+          employees.append(employee)
+        end
+      }
+    end
+    if employee['hireDate'].present? &&
+          [day_before_yesterday, yesterday, today, tomorrow, day_afer_tomorrow].any? { |d| 
+          if d.include? employee['hireDate']
+            employees.append(employee)
+          end
+          }
+    end
+    time_off.map { |e|
+      if e['employeeId'].to_s.include? employee['id']
+        employees.append(employee)
```

Comment:
So you are appending employees with recent/upcoming hire/birthdays. What happens to this array once you get all the employees? Does this reset each day?